src/main/java/uk/gov/hmcts/cmc/claimstore/services/search/CaseRepository.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.cmc.claimstore.services.search;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.claimstore.repositories.CaseDBI;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class CaseRepository {
+
+    private final CaseDBI caseDBI;
+
+    public CaseRepository(CaseDBI caseDBI) {
+        this.caseDBI = caseDBI;
+    }
+
+    public List<Claim> getBySubmitterId(String submitterId, String authorisation) {
+        return caseDBI.getBySubmitterId(submitterId);
+    }
+
+    public Optional<Claim> getClaimByExternalId(String externalId, String authorisation) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "authorisation". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1172)